src/ArchitectureSnifferFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace ArchitectureSniffer;
+
+use phpDocumentor\Reflection\DocBlock;
+use phpDocumentor\Reflection\DocBlockFactory;
+
+class ArchitectureSnifferFactory
```

Comment:
Why do we need this?

Reply:
Yes. I will delete trait and factory files.

Reply:
Deleted files